src/main/resources/static/app/request/edit-request.html

```diff
@@ -871,9 +871,9 @@ <h4 class="h4 h4-palga">{{'Final checks before data delivery'|translate}}</h4>
                 </div> <!--end modal-body-->
 
                 <div class="modal-footer">
-                    <a id="cancel-new-request" class="btn btn-default pull-left req-edit-button" ng-click="cancel(request);">
+                    <button id="cancel-new-request" class="btn btn-default pull-left req-edit-button" ng-click="cancel(request);">
```

Comment:
Is this needed? The reason why it's an `a` and not `button` is that the button elements are highlighted by using the tab key and the link element not. This is to try prevent accidentally using the 'Cancel' button and losing your data.